src/styles/blocks/star.scss

```diff
@@ -0,0 +1,22 @@
+.content__stars{
+  margin: 16px 16px 0 16px;
+  display: flex;
+  justify-content: space-between;
+}
+.stars{
+  display: flex;
+}
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+.stars--4 .stars__star:nth-child(-n + 4){
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
The selector `.stars--4 .stars__star:nth-child(-n + 4)` is specific to a 4-star rating. If you plan to support different star ratings (e.g., 3 stars, 5 stars), consider using a more scalable approach, such as dynamically adding a class based on the rating or using a data attribute to control the number of active stars.